compute/client.go

```diff
@@ -133,6 +147,47 @@ func (c *Client) executeRequest(method, path string, body interface{}) (*http.Re
 	return nil, oracleErr
 }
 
+// Allow retrying the request until it either returns no error,
+// or we exceed the number of max retries
+func (c *Client) retryRequest(req *http.Request) (*http.Response, error) {
+	// Double check maxRetries is not nil
+	var retries int
+	if c.maxRetries == nil {
```

Comment:
Final check. But also during the test, we don't call `NewComputeClient`, so we don't have that check there.